services/src/main/scala/cromwell/pipeline/service/ProjectService.scala

```diff
@@ -5,6 +5,7 @@ import java.util.UUID
 import cromwell.pipeline.datastorage.dao.repository.ProjectRepository
 import cromwell.pipeline.datastorage.dto._
 import cromwell.pipeline.service.Exceptions.{ ProjectAccessDeniedException, ProjectNotFoundException }
+import cromwell.pipeline.datastorage.dto.{ Project, ProjectAdditionRequest, ProjectId }
```

Comment:
Why do we need this import if we already importing everything on line 6?

Reply:
Fixed